src/services/MoxfieldService.ts

```diff
@@ -0,0 +1,47 @@
+import axios from "axios";
+import { useCallback } from "react";
+
+const useGetMoxfieldProfile = () => {
+    const endpoint = "https://chatterfang.onrender.com/moxfield/";
+
+    return useCallback(async (moxfieldId: string) => {
+        try {
+            const _res = await axios.get<MoxfieldResponse>(endpoint + moxfieldId, {
+                headers: { "Content-Type": "application/json" }
+            });
+            return _res.data;
+        } catch (err) {
+            console.log(err);
+        }
+        return undefined;
+    }, []);
+};
+
+export const MoxfieldService = {
+    useGetMoxfieldProfile
+};
+
+export type MoxfieldResponse = {
```

Comment:
can we move this into "service/types"? 

you can now see the reason we have a service/types-- because that's what's getting returned by the service